actions/storage.js

```diff
@@ -79,3 +79,21 @@ export function actionOnItemsWithPrefix(prefix, action) {
         return {data: false};
     };
 }
+
+export function storageRehydrate(incoming) {
+    return async (dispatch, getState) => {
+        if (incoming.storage) {
+            let storage = {}
+            try {
+                storage = JSON.parse(incoming.storage)
+            } catch (err) {
+                if (process.env.NODE_ENV !== 'production') console.warn(`Error rehydrating data for key "storage"`, err)
+            }
+            dispatch({
+                type: StorageTypes.STORAGE_REHYDRATE,
+                data: storage
+            });
+        }
+        return {data: false};
```

Comment:
why is this returning false?

Reply:
Sorry, was based on the previous code, and the previous code was an error on my understanding of the async actions expected returns. I fixed this and the previous actions.